src/mpyl/reporting/targets/github.py

```diff
@@ -78,6 +85,26 @@ def _get_pull_request(self, repo: GithubRepository, run_properties: RunPropertie
             return get_pr_for_branch(repo, current_branch)
         return None
 
+    def _update_pr(self) -> Callable[[PullRequest, RunResult], None]:
+        if self.update_strategy == GithubUpdateStategy.COMMENT:
+            return self._change_pr_comment
+        return self._change_pr_body
+
+    def _change_pr_body(self, pull_request: PullRequest, results: RunResult):
+        current_body = (pull_request.body.split("----")[0] if pull_request.body else "") + "\n----\n"
```

Comment:
Maybe you can extract `"----"` to a constant

Reply:
How would you unit test this though? Since this just wraps the github API. I can mock that, but then the test is not much

Reply:
I mean _just_ the replacement logic with a string. To make sure that you're not adding a newline with every update for example.